packages/core/js-sdk/src/admin/api-key.ts

```diff
@@ -0,0 +1,104 @@
+import { FindParams, HttpTypes, PaginatedResponse } from "@medusajs/types"
+import { Client } from "../client"
+import { ClientHeaders } from "../types"
+
+export class ApiKey {
+  private client: Client
+  constructor(client: Client) {
+    this.client = client
+  }
+
+  async list(queryParams?: FindParams, headers?: ClientHeaders) {
```

Comment:
Should we not have a separate type for the query params here? I would expect you can do more filters then just FindParams